### PR TITLE
docs: remove misleading redirect

### PR DESCRIPTION
**Pull request by garrett361**
## Description

Currently, the "TensorFlow Keras Fashion MNIST Tutorial" page gets automatically re-directed to the "PyTorch MNIST Tutorial" page, making the former tutorial inaccessible.  Delete the re-direct and restore user access to the TF tutorial.  

## Test Plan

Go to the [Tutorials](https://docs.determined.ai/latest/tutorials/index.html) page, click on the "TensorFlow Keras Fashion MNIST Tutorial" and make sure it leads you to the correct tutorial.

## Checklist
- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.
- [ ] If modifying `/webui/react/src/shared/` verify `make -C webui/react test-shared` passes.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by sirredbeard — APPROVED** [review]
LGTM
